### ISSUE TITLE
CUDA?

### ISSUE THREAD

**Issue by iwoolf** [bugbakery/audapolis#468]
Hi, I’m a podcaster trying out audapolis on Ubuntu  LTS. Is there a way to get it to use my nvidia 3060 to speed it up with CUDA? nvidia-smi shows its not using the gpu. My first attempt caused an oom that killed all my apps. I’m now running it alone, but its still very slow.

**Comment by yburkouski** [comment]
Agreed, mining boom is over 1.5 year ago, is I understand it's 1 line to use cuda in a pytorch
